package.json

```diff
@@ -45,8 +45,8 @@
     "md5": "^2.0.0",
     "minimatch": "~1.0.0",
     "q": "~1.0.1",
-    "steal": "0.13.X",
-    "steal-tools": "0.13.X",
+    "steal": "0.16.6",
+    "steal-tools": "0.16.6",
```

Comment:
After adding engineDependencies for Node 6 and 7 the [build still fails](https://travis-ci.org/pmgmendes/documentjs/jobs/225398042) due 6 failing tests. The troubleshooting points me to steal.js, getScriptOptions(), line 6980
```
...
var optionName =
	camelize( attr.nodeName.indexOf("data-") === 0 ?
		attr.nodeName.replace("data-","") :
		attr.nodeName );
...
```
not sure though..
Adding the caret range specifier causes 1 test to fail in the [node 0.10 build](https://travis-ci.org/pmgmendes/documentjs/jobs/225398040). Still didn't got into it.